src/rev_ai/apiclient.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using rev.ai
+"""
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+import os
+import requests
+import logging
+import json
+from models import (
+    Job,
+    JobSubmitOptions,
+    Account,
+    Transcript
+)
+
+LOG = logging.getLogger(__name__)
+
+class RevAiAPIClient:
+    """Client which implements rev.ai API"""
+
+    version = "v1"
+    """Default version of rev.ai"""
```

Comment:
* i've never seen a comment below the relevant line
* usually don't use triple quote unless quoting a multiline string or method docstring. I would have used `#`